docs/KNOWN_ORGANIZATIONS.md

```diff
@@ -0,0 +1,8 @@
+# Known organizations
+
+Want to have your organization's swanky cool name and logo show up in the Client instead of an identicon? Just follow these steps!
```

Comment:
```suggestion
Want to have your organization’s swanky cool name and logo show up in the Client instead of an identicon? Just follow these steps!
```